### PR TITLE
Add Night mode

### PR DESCRIPTION
**Pull request by smrm-dev**

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -22,7 +22,7 @@ if (flutterVersionName == null) {
 }
 
 apply plugin: 'com.android.application'
-apply plugin: 'com.google.gms.google-services'
+// apply plugin: 'com.google.gms.google-services'
```
Comment (DeMonkeyCoder) [inline android/app/build.gradle:25]: Don't do changes unrelated to PR's description. If you have build problems, file another issue

File: lib/main.dart [hunk 2]
```diff
@@ -140,7 +153,7 @@ class _HomePageState extends State<HomePage> {
         maxLines: null,
         decoration: new InputDecoration(
```
Comment (DeMonkeyCoder) [inline lib/main.dart:154]: `new` is not needed In new versions of dart. @alifarazz should handle CI for avoiding these kind of problems.

File: pubspec.lock [hunk 3]
```diff
@@ -7,49 +7,42 @@ packages:
       name: archive
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.0.13"
+    version: "2.0.11"
```
Comment (DeMonkeyCoder) [inline pubspec.lock:10]: Upgrading or downgrading pub is unrelated to your PR.
You should provide description for these changes in your PR.

Besides, use flutter channel master for running this application.

**Review by DeMonkeyCoder — CHANGES_REQUESTED** [review]
Thanks for your PR.
I think you should make some changes to your PR and send PR from another branch (like dark-theme).
Also write fixes #15 in your PR's description.
See other open-source contributions as a reference. I think #1 is a good one to see.
Thanks!

**Comment by DeMonkeyCoder** [comment]
The build problem is fixed. Add my repository as upstream with `git remote add upstream https://github.com/alimahdiyar/noteif.git` rebase with master with `git pull upstream master --rebase` in your new PR

File: lib/main.dart (hunk 2 above)
Comment (smrm-dev) [reply lib/main.dart:154]: I did not use the keyword ```new``` in my code, it was used by previous developers.

Comment (DeMonkeyCoder) [reply lib/main.dart:154]: was my mistake.
You're right.
@alifarazz should take care of this.

**Comment by DeMonkeyCoder** [comment]
Working on #24.
Closed.

**Pull request closed without merging**